### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by jaspreet-singh-sahota**
Feature of the project:-
- On page load, it with show the weather report of the user's location
- User can search weather report of different cities
- User can toggle temperature between Calvin or Fahrenheit

Note:- I have added dummy data at the bottom, it is not functional. This weather API does not provide enough information that I wanted. Hope maybe in future I will find the desired API. Meanwhile, I am leaving it like this, please ignore it.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        } catch (err) {
+          e.target.value = '';
+          placeholderValue.placeholder = 'Enter a valid city name';
+          placeholderValue.classList.add('error');
+        }
+      });
+
+      updateData(data);
+      icon.add('icon9', iconWeatherCheck(data.weather[0]));
+    });
```
Comment (shloch) [inline src/index.js]: - we are required to not mix LOGIC and DOM in this project
The weather-fetching-code appears to be the main logic here, look for a way to get that logic in a separate file, such that it takes a certain city in parameter and returns whether information. You can now use this function in a DOM module to play with the data

[ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png)]

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @jaspreet-singh-sahota , 


The final output is awesome......Really great job

### [changes required]

Howerver, i want you to polish a little one aspect cited below :

File: src/index.js (hunk 1 above)
Comment (jaspreet-singh-sahota) [reply src/index.js]: Hey Shloch, thanks for the suggestion, now I can see my code base is even cleaner. And thanks for your feedback on the project.

**Review by suretrust — APPROVED** [review]
Hi @jaspreet-singh-sahota, 👋

### Great job implementing the suggested changes. 🙌 Let's get this merged! 👍 

_Good luck with your future projects and happy coding!_ 🎉

**Pull request merged**